### ISSUE TITLE
Custom Assistive Text removed when image is masked into a shape

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5653]
## Bug Description

Originally reported by @kkalarickal 

When an image that DID NOT contain assistive text is updated with custom assistive text in the stories editor, and then  is masked into a shape, the custom assistive text is replaced with an empty value, resulting in a Checklist recommendation.

## Expected Behaviour

Assistive text entered about an image or already existing in an image should NOT be overwritten just because the user moved the image into a shape mask.

It’s possible that masking introduces a new element in the page that is no longer the original image. But clearly the intent of the user was to retain the image and its contents. Assistive text from the image should also be retained into the new masked element.

## Steps to Reproduce

- Create a web story and drop an image into a page which does not have assistive text
- Note that the checklist shows a recommendation to add assistive text
- Update the assistive text section of the image with some text
- Note that the checklist does not show the recommendation to add assistive text
- Now place a shape in the same page
- Drag the image into the shape so that it is masked by the shape
- Note that the checklist once again shows the recommendation to add assistive text

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by csossi** [comment]
Verified in QA
